dora/core/server/master/src/main/java/alluxio/master/scheduler/Scheduler.java

```diff
@@ -463,28 +299,21 @@ private void processJobs() {
     if (Thread.currentThread().isInterrupted()) {
       return;
     }
-    mJobToRunningTasks.forEach((k, v) -> processJob(k.getDescription(), k));
+    mJobToRunningTasks.forEach((k, v) -> processJob(k));
   }
 
-  private void processJob(JobDescription jobDescription, Job<?> job) {
-    if (!job.isRunning()) {
-      try {
-        LOG.debug("Job:{}, not running, updating metastore...", MoreObjects.toStringHelper(job)
-            .add("JobId:", job.getJobId())
-            .add("JobState:", job.getJobState())
-            .add("JobDescription", job.getDescription()).toString());
-        mJobMetaStore.updateJob(job);
```

Comment:
@lucyge2022 I remove this update job since I believe we already record journal everytime we have status change

Reply:
sounds good